src/migrations/0002.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Cronbeat\Migrations;
+
+use Cronbeat\BaseMigration;
+use Cronbeat\Logger;
+
+/**
+ * @noinspection PhpUnused
+ */
+class Migration0002 extends BaseMigration {
```

Comment:
@jetbrains-junie during migration in docker, using web interface, I get this error

> Fatal error: Uncaught Error: Class "Cronbeat\MigrationHelper" not found in /var/www/html/controllers/MigrateController.php:50 Stack trace: #0 /var/www/html/controllers/MigrateController.php(24): Cronbeat\Controllers\MigrateController->processMigration() #1 /var/www/html/index.php(67): Cronbeat\Controllers\MigrateController->doRouting() #2 {main} thrown in /var/www/html/controllers/MigrateController.php on line 50